src/components/app/AppViewRouter.js

```diff
@@ -81,7 +82,8 @@ class AppViewRouterImpl extends PureComponent<AppViewRouterProps> {
         return <ProfileLoaderAnimation />;
       case 'FATAL_ERROR': {
         const message =
-          ERROR_MESSAGES[dataSource] || "Couldn't retrieve the profile.";
+          ERROR_MESSAGES_L10N_ID[dataSource] ||
+          'AppViewRouter--error-message-public';
         let additionalMessage = null;
```

Comment:
For a future PR: this `additionalMessage` needs some localisation too, but how to do it isn't obvious. Probably this is a case where we'd need `getString` from https://github.com/projectfluent/fluent.js/wiki/withLocalization...